packages/core/src/components/TextArea/Proofreader/ErrorMenu.tsx

```diff
@@ -16,6 +16,17 @@ export default class ErrorMenu extends React.Component<Props> {
     this.props.onReplaceText(this.props.error, replacement);
   };
 
+  private replacementPhrase = (replacement: string) => {
```

Comment:
is this `private` needed? usually just added for handlers

Reply:
It doesn't seem necessary, but to me (admittedly neither a TS nor Lunar expert), I'm not sure there's harm in adding it to encapsulate implementation within this component. Can remove if convention and style disagree.

Reply:
not a blocker! did a rough scan of lunar and it is mostly in front of handlers, but some other methods too. seems fine!